### PR TITLE
Ops 1215 agreements tabs

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Added tabs to Agreement List page: All Agreements, My Agreements

## Issue

https://github.com/HHS/OPRE-OPS/issues/1215

## How to test

1. Navigate to the Agreement List page.
2. Verify the default view lists all Agreements.
3. Add yourself as a team member to one or more Agreement.
4. Click the `My Agreements` tab and verify your Agreements are listed.

## Screenshots

image

image

### CODE REVIEW

File: frontend/src/pages/agreements/list/AgreementsTable.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Agreement table.
+ * @param {Object} props - The component props.
+ * @param {Object[]} props.agreements - Array of Agreement to display in the table.
+ * @returns {ReactNode} The rendered component.
+ */
+export const AgreementsTable = ({ agreements }) => {
+    return (
+        <table className="usa-table usa-table--borderless width-full ">
```
Comment (fpigeonjr) [inline frontend/src/pages/agreements/list/AgreementsTable.jsx:12]: wonder if we need like a fallback option if no agreements are found?

**Review by fpigeonjr — CHANGES_REQUESTED** [review]
🔥Ran in `localdev` and looks great. Made some minor suggestions.

File: frontend/src/components/UI/TabsSection/TabsSection.module.scss [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    text-decoration-color: #005ea2;
+    color: #005ea2;
+    outline: none;
+}
+
+.listItemNotSelected {
+    display: inline-block;
+    padding: 5px;
+    text-decoration-line: none;
+    color: #565c65;
```
Comment (fpigeonjr) [inline frontend/src/components/UI/TabsSection/TabsSection.module.scss:27]: uswds color is `text-base-dark`

File: frontend/src/pages/agreements/list/AgreementsTable.jsx (hunk 1 above)
Comment (johndeange) [reply frontend/src/pages/agreements/list/AgreementsTable.jsx:12]: Added this.

Comment (fpigeonjr) [reply frontend/src/pages/agreements/list/AgreementsTable.jsx:12]: Don't think we even need to render the table if there aren't any agreements. Maybe something like: 
```js
{
  agreements.length > 0 ?  (
    // render table
  ) : (
   // no agreements
  )
}
```

**Review by fpigeonjr — CHANGES_REQUESTED** [review]
thanks for taking care of this. One last suggestion.

**Review by fpigeonjr — APPROVED** [review]
👍🏿thanks @johndeange,  Looks good!

**Pull request merged**